### PR TITLE
REWARDS: add info to reward events, totalAwarded 

### PR DESCRIPTION
**Pull request by vistra**
- Added `withdrawnFees`, `withdrawenBootstrap`. Emitting `totalAwarded` in assignment events and `totalWithdrawn` in withdraw events.
- Adding `feesPerMember`, `bootstrapPerMember`, `certification` to assignment events
- Added `totalClaimed` and certification to `getFeesAndBootstrapData`
- Added `stake` to `delegatorStakingRewardsAssigned events`

### CODE REVIEW

File: contracts/FeesAndBootstrapRewards.sol [hunk 1]
```diff
@@ -95,7 +97,8 @@ contract FeesAndBootstrapRewards is IFeesAndBootstrapRewards, ManagedContract {
         updateGuardianFeesAndBootstrap(guardian);
         uint256 amount = feesAndBootstrap[guardian].bootstrapBalance;
         feesAndBootstrap[guardian].bootstrapBalance = 0;
-        emit BootstrapRewardsWithdrawn(guardian, amount);
+        feesAndBootstrap[guardian].withdrawnBootstrap = feesAndBootstrap[guardian].withdrawnBootstrap.add(amount);
```
Comment (OdedWx) [inline contracts/FeesAndBootstrapRewards.sol:100]: can assign the updated withdrawnBootstrap  in a memory variable

File: contracts/FeesAndBootstrapRewards.sol [hunk 2]
```diff
@@ -105,7 +108,9 @@ contract FeesAndBootstrapRewards is IFeesAndBootstrapRewards, ManagedContract {
 
         uint256 amount = feesAndBootstrap[guardian].feeBalance;
         feesAndBootstrap[guardian].feeBalance = 0;
-        emit FeesWithdrawn(guardian, amount);
+        feesAndBootstrap[guardian].withdrawnFees = feesAndBootstrap[guardian].withdrawnFees.add(amount);
```
Comment (OdedWx) [inline contracts/FeesAndBootstrapRewards.sol:111]: can assign the updated withdrawnFees in a memory variable

File: contracts/spec_interfaces/IStakingRewards.sol [hunk 3]
```diff
@@ -5,8 +5,8 @@ pragma solidity 0.6.12;
 /// @title Staking rewards contract interface
 interface IStakingRewards {
 
-    event DelegatorStakingRewardsAssigned(address indexed delegator, uint256 amount, uint256 totalAwarded, address guardian, uint256 delegatorRewardsPerToken);
-    event GuardianStakingRewardsAssigned(address indexed guardian, uint256 amount, uint256 totalAwarded, uint256 delegatorRewardsPerToken, uint256 stakingRewardsPerWeight);
+    event DelegatorStakingRewardsAssigned(address indexed delegator, uint256 amount, uint256 totalAwarded, address guardian, uint256 delegatorRewardsPerToken, uint256 delegatorRewardsPerTokenDelta);
+    event GuardianStakingRewardsAssigned(address indexed guardian, uint256 amount, uint256 totalAwarded, uint256 delegatorRewardsPerToken, uint256 stakingRewardsPerWeight, uint256 stakingRewardsPerWeightDelta, uint256 delegatorRewardsPerTokenDelta);
```
Comment (OdedWx) [inline contracts/spec_interfaces/IStakingRewards.sol:9]: would be nicer to reorder:
event GuardianStakingRewardsAssigned(address indexed guardian, uint256 amount, uint256 totalAwarded, uint256 delegatorRewardsPerToken, uint256 delegatorRewardsPerTokenDelta, uint256 stakingRewardsPerWeight, uint256 stakingRewardsPerWeightDelta);

Comment (vistra) [reply contracts/spec_interfaces/IStakingRewards.sol]: Done

File: contracts/FeesAndBootstrapRewards.sol (hunk 2 above)
Comment (vistra) [reply contracts/FeesAndBootstrapRewards.sol]: Done

File: contracts/FeesAndBootstrapRewards.sol (hunk 1 above)
Comment (vistra) [reply contracts/FeesAndBootstrapRewards.sol]: Done

**Review by OdedWx — APPROVED** [review]

**Pull request merged**